test/config/pebble-config.json

```
@@ -4,6 +4,7 @@
     "certificate": "test/certs/localhost/cert.pem",
     "privateKey": "test/certs/localhost/key.pem",
     "httpPort": 5002,
-    "tlsPort": 5001
+    "tlsPort": 5001,
+    "ocspResponderURL": "test"
```

Can you make this value `""` in the example config instead? I think it makes the configuration field discoverable by those that need it, but prevents the default configuration from including a non-sense URL in the issued certificates.
```suggestion
    "ocspResponderURL": ""
```

---

Ah yes, I was about to set this as empty, it was a mistake during the push.